app/assets/javascripts/admin/bulk_product_update.js.coffee

```diff
@@ -108,9 +108,13 @@ angular.module("ofn.admin").controller "AdminProductEditCtrl", ($scope, $timeout
     $scope.categoryFilter = "0"
     $scope.importDateFilter = "0"
 
+  confirm_unsaved_changes = () -> (DirtyProducts.count() > 0 and confirm(t("unsaved_changes_confirmation"))) or (DirtyProducts.count() == 0)
```

Comment:
I'd break line after -> so that the line is shorter and declaration and implementation are in diff lines.
same for the editProductUrl method

Reply:
Done :)